parsers/mcmaster-parser-2.js

```diff
@@ -19,6 +19,53 @@ function convertTo24Hour(time) {
 }
 
 /**
+ * Takes any date and turns it into the correct format, so javascript gets the right date.
+ * MM/DD/YYYY is the format the javascript object reads or YYYY/MM/DD
+ * @param {String} startDate
+ * @param {String} endDate
+ * @return {String Array} [startDate,endDate]  woth format MM/DD/YYYY or YYYY/MM/DD
+ */
+function fixDateFormat(sem_start, sem_end) {
+    if (sem_start.indexOf('-') > -1) {
+        return [sem_start,sem_end]
```

Comment:
need spaces between array values, do this everywhere else as well (including comment)